### DISCUSSION TITLE
Could auth.signInWithSSO accept queryString login_hint?

### DISCUSSION THREAD

**Discussion by brianq** [supabase/supabase#30506] [Feature Requests]
I have a site where I've implemented a login page with a rudimentary domain discovery (based on the user's email address domain). 

Retrieving the `providerId` for the given email domain and calling  `auth.signInWithSSO()` works great, however the user is taken to their (Azure) login page where they have to re-enter the email address again. 

Might it be possible to pass a `queryString.login_hint` parameter in `options` (like how it's done with `signInWithOAuth`) or is there any other way of doing this?

**Comment by chris417** [comment]
+1 - be great to have this.
